demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -151,23 +150,32 @@ def prepare_hooks(
             "all_files": self.all_files,
             "input_mode": self.input_mode,
         }
-        PyclnHook(**hooks["pycln"], **kwargs).prepare_hook(PYTHONPATH)
-        RuffHook(**hooks["ruff"], **kwargs).prepare_hook(
+        PyclnHook(**hooks.pop("pycln"), **kwargs).prepare_hook(PYTHONPATH)
```

Comment:
these will still fail if the hooks arent present. We should enable not having these hooks in the config. Pop with None as default return